src/views/modules/NavBar.vue

```diff
@@ -1,57 +1,54 @@
 <template>
   <nav>
     <ul>
-      <li 
-        v-for="item in items" 
-        :key="item.id" 
-        :class="{ active: item.id === activeItemId }" 
+      <li
+        v-for="item in items"
+        :key="item.id"
+        :class="{ active: item.id === activeItemId }"
       >
         <router-link :to="item.link">
           {{ item.text }}
         </router-link>
       </li>
-      <li @click="">
-        <div class="sidebar_logo">
-          <div class="sidebar_logo_items"></div>
-          <div class="sidebar_logo_items"></div>
-          <div class="sidebar_logo_items"></div>
-        </div>
+      <li>
+        <Logo @click="$emit('toggle')" />
       </li>
     </ul>
   </nav>
 </template>
 
 <script>
-import  Sidebar  from "./Sidebar.vue";
-import { defineComponent, reactive } from 'vue';
-
+import Logo from "./SidebarLogo.vue";
```

Comment:
Could you rename the file Logo ( SidebarLogo) instead of naming it with BurgerMenuButton or IconButton etc?

Cause logo in this context is not correct I assume.

Reply:
Okey, thanks!